### PR TITLE
F# documentation completion item fix

### PR DESCRIPTION
**Pull request by TIHan**
Resolves this: https://github.com/dotnet/interactive/issues/626

It does not add any semantic/syntax highlights on the text for the tooltip. We should do that in a separate PR.

Another note: "FSharp.Core.xml" must be in the published output of "dotnet-interactive.csproj" in order to see "FSharp.Core" library documentation in notebooks. See here: https://github.com/dotnet/interactive/issues/842

### CODE REVIEW

**Comment by TIHan** [comment]
This is ready

**Comment by jonsequitur** [comment]
Got tests?

**Review by KevinRansom — APPROVED** [review]

File: src/Microsoft.DotNet.Interactive.FSharp.Tests/KernelTests.fs [hunk 1]
```diff
@@ -44,6 +44,24 @@ type KernelTests() =
             
             texts
 
+    let getCompletions (line : int) (column : int) (code : #seq<string>)  =
+        let code = String.concat "\r\n" code
```
Comment (jonsequitur) [inline src/Microsoft.DotNet.Interactive.FSharp.Tests/KernelTests.fs:48]: We probably won't want the `\r` on non-Windows OSs, yeah?

Comment (TIHan) [reply src/Microsoft.DotNet.Interactive.FSharp.Tests/KernelTests.fs:48]: Probably not, good catch

**Review by jonsequitur — APPROVED** [review]

**Pull request merged**